scripts/geojson/index.js

```diff
@@ -0,0 +1,134 @@
+const fs = require('fs'),
+      argv = require('minimist')(process.argv.slice(2)),
+      request = require('request'),
+      //extract = require('extract-zip'),
+      AdmZip = require('adm-zip'),
+      xmldom = new (require('xmldom').DOMParser)(),
+      tj = require('togeojson'),
+      mkdirp = require('mkdirp'),
+      path = require('path'),
+      { URL } = require('url');
+
+(function main(argv) {
+
+  const countryCode = argv._[0]
+  console.info(`# Starting ... ${countryCode}`)
+
+  const config = {
+    path: {
+      out: argv._[1] || '/output',
+      tmp: argv._[2] || '/tmp'
+    },
+    url: {
+      base: 'http://biogeo.ucdavis.edu/data/gadm2.8/kmz/'
+    }
+  }
+
+  Promise.resolve()
+    .then(() => processCountry(countryCode, 0, config))
+    .then(() => processCountry(countryCode, 1, config))
+    .then(function() {
+      console.log("# All done!!")
+    })
+    .catch(function(err) {
+      console.error(`ERROR: ${err}`)
+    })
+})(argv)
```

Comment:
Can we display a usage message if no arguments are given and support the `-h` and `--help` command line options?